templates/package.json

```diff
@@ -6,6 +6,18 @@
     "type": "git",
     "url": "https://github.com/ubilabs/{{project-name}}.git"
   },
+  "precommit": {
+    "run": "test",
+    "template": ".gitcommit.template"
+  },
   "scripts": {
+    "build": "npm run clean && NODE_ENV=production webpack -p",
+    "changelog": "conventional-changelog-generator && $EDITOR -w CHANGELOG.md",
```

Comment:
Does that work with the npm scripts? Do the scripts wait for the editor to finish? And is every editor able to do this?